.github/CODEOWNERS

```diff
@@ -16,7 +16,7 @@
 /sdk/azcore/                  @rickwinter @chlowell @jhendrixMSFT @richardpark-msft
 
 # PRLabel: %Azure.Identity
-/sdk/azidentity/              @chlowell @jhendrixMSFT @rickwinter
+/sdk/azidentity/              @chlowell @jhendrixMSFT @rickwinter @Azure/azsdk-identity
```

Comment:
This addition is being flagged as an error.  Does this team exist?

Reply:
I think it may be missing a couple of the access requirements we have in our automation (https://github.com/Azure/azure-sdk-tools/issues/4487#issuecomment-1614887191). We will be trying to set that up this week.

Reply:
The team exists and is publicly visible, so I guess it lacks write access to the repo. Following Jim's comment, I suppose the solution is to make Azure/azsdk-identity a member of [azure-sdk-write](https://github.com/orgs/Azure/teams/azure-sdk-write). Do you know who's able to do that?

Reply:
@chlowell we have an active thread on this in Teams [here](https://teams.microsoft.com/l/message/19:59dbfadafb5e41c4890e2cd3d74cc7ba@thread.skype/1690220936782?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1690220936782&teamName=Azure%20SDK&channelName=Engineering%20System%20%F0%9F%9B%A0%EF%B8%8F&createdTime=1690220936782).